### PR TITLE
Document auth controller

### PR DESCRIPTION
**Pull request by ballinette**
+ refactor MD filenames

### CODE REVIEW

**Comment by benoitvidis** [comment]
+1

**Comment by stafyniaksacha** [comment]
+1

File: source/includes/_authController.md [hunk 1]
```diff
@@ -0,0 +1,75 @@
...
+
+## login
+
+<section class="rest"></section>
+>**URL:** `http://kuzzle:7512/api/_login`  
+>**Method:** `POST`
+>**Body:**
+
+<section class="rest"></section>
+```litcoffee
```
Comment (stafyniaksacha) [inline source/includes/_authController.md]: Why litcoffe and not javascript here ?

Comment (benoitvidis) [inline source/includes/_authController.md]: The Rouge parser is quite strict when dealing with syntax check.
When a curly brace is open without a var statement in javascript, it considers the inside as being a code snippet and not an object, and displays some error css.
The litcoffe produces a nicer output, even if semantically, it is not coffee script at all.

cf http://rouge.jneen.net/pastes/0bO vs http://rouge.jneen.net/pastes/LgZ vs http://rouge.jneen.net/pastes/nmq

**Pull request merged**